src/components/publiq-ui/Alert.js

```diff
@@ -22,21 +22,22 @@ const Alert = ({
   className,
   ...props
 }) => {
+  // TODO: box props (positioning) doesn't seem to work
```

Comment:
Nit: It's usually best to refrain from adding todo's in code and create a ticket instead so it doesn't get lost in all the other work. But since this app is still very actively being developed and optimized I trust it will get picked up 😉

Reply:
Good to know 😸

Reply:
https://jira.uitdatabank.be/browse/III-3676